cfme/containers/provider/__init__.py

```diff
@@ -269,6 +270,18 @@ def num_image_registry(self):
     def num_image_registry_ui(self):
         return int(self.get_detail("Relationships", "Image Registries"))
 
+    def pods_per_ready_status(self):
+        """Grabing the Container Statuses Summary of the pods from API"""
+        #  TODO: Add later this logic to wrapanapi
+        entities_j = self.mgmt.api.get('pod')[1]['items']
```

Comment:
why `_j` on this variable name? nitpicking, but it doesn't seem to make sense, `entities` would be sufficient.